apps/frontend/core-dapp/src/features/simulator/SimulatorCard.tsx

```diff
@@ -81,71 +60,86 @@ const MarketNotAvailable = styled.span`
-    isDirty: isEntryPriceDirty,
-    reset: resetEntryPrice,
-    setValue: setEntryPrice,
-    value: entryPrice,
+  entryDistance: {
+    isDirty: isEntryDistanceDirty,
+    reset: resetEntryDistance,
+    setValue: setEntryDistance,
+    value: entryDistance,
   },
-  exitPrice: {
-    isDirty: isExitPriceDirty,
-    reset: resetExitPrice,
-    setValue: setExitPrice,
-    value: exitPrice,
+  exitDistance: {
+    isDirty: isExitDistanceDirty,
+    reset: resetExitDistance,
+    setValue: setExitDistance,
+    value: exitDistance,
   },
-  marketResolved,
+  HeaderComponent,
+  marketResolved = false,
+  onChangeDirection,
   tradeSize,
 }) => (
-  <Wrapper>
-    <Header>
-      <Flex gap={4}>
-        <Title>Profit Simulator</Title>
-        <Tooltip overlay={<SimulatorDefinition />}>
-          <Icon name="info-outlined" color="neutral5" height="14" width="14" />
-        </Tooltip>
-      </Flex>
-      {(isEntryPriceDirty || isExitPriceDirty) && (
-        <ResetButton
-          onClick={(): void => {
-            resetEntryPrice()
-            resetExitPrice()
-          }}
-        >
-          Reset
-        </ResetButton>
-      )}
-    </Header>
+  <Wrapper className={className}>
+    <HeaderComponent
+      resetButton={
+        <>
+          {(isEntryDistanceDirty || isExitDistanceDirty) && (
+            <ResetButton
+              onClick={(): void => {
+                resetEntryDistance()
+                resetExitDistance()
+              }}
+            >
+              Reset
+            </ResetButton>
+          )}
+        </>
+      }
+    />
+    {onChangeDirection && (
+      <DirectionRadio direction={direction} onChangeDirection={onChangeDirection} size="sm" />
+    )}
     <SimulatorWrapper>
       {marketResolved ? (
         <MarketNotAvailable>Market Ended</MarketNotAvailable>
       ) : (
         <Simulator
           data={data}
           direction={direction}
-          entryPrice={entryPrice}
-          exitPrice={exitPrice}
-          isEntryPriceDirty={isEntryPriceDirty}
-          isExitPriceDirty={isExitPriceDirty}
-          onChangeExit={setExitPrice}
-          onChangeEntry={setEntryPrice}
+          entryDistance={data ? entryDistance : undefined}
+          exitDistance={data ? exitDistance : undefined}
+          isEntryDistanceDirty={isEntryDistanceDirty}
+          isExitDistanceDirty={isExitDistanceDirty}
+          onChangeExitDistance={setExitDistance}
+          onChangeEntryDistance={setEntryDistance}
         />
       )}
     </SimulatorWrapper>
     {!marketResolved && (
       <SimulatorSummary
         direction={direction}
         payoutRange={data?.payoutRange}
-        entryPrice={entryPrice}
-        exitPrice={exitPrice}
+        entryPrice={mapDistanceToPrice({
+          data,
+          direction,
+          distance: entryDistance,
+        })}
+        exitPrice={mapDistanceToPrice({
+          data,
+          direction,
+          distance: exitDistance,
+        })}
```

Comment:
After removing `direction` in `mapDistanceToPrice`: 

```suggestion
        entryPrice={mapDistanceToPrice({ data, distance: entryDistance })}
        exitPrice={mapDistanceToPrice({ data, distance: exitDistance })}
```